api/files/serializers.py

```diff
@@ -232,6 +232,8 @@ def update(self, instance, validated_data):
 
         for attr, value in validated_data.items():
             if attr == 'checkout':
+                if value == '':
```

Comment:
The problem is rather 2 lines down, `self.checkout = checkout` results in `
TypeError: Value must be StoredObject, tuple, or None`  if "checkout" is passed through as `''` - I'll change where this conversion happens though for that line instead of in the serializer